math_science_reqs/18202.md

```diff
@@ -0,0 +1,80 @@
+|  Exams   |     5      |
+| Quizzes  |     2      |
+
+Required core corequisite for ECE majors. This course is a broad introduction to the branches of mathematics applied
+in electrical engineering, including: linear algebra, differential equations, and some complex/vector calculus. While it is
+not explicitly a prerequisite, many core ECE courses, especially in the signal processing/circuits paths, will reference 
+topics learned in 18-202. 
+
+Due to its breadth, the course moves relatively briskly through topics, relying on students to stay on track through
+recitation and assigned readings. Topic lectures are also supplemented by "Idea lectures" which
+demonstrate real-world applications of the topics being taught. While these are interesting to listen to,
+students shouldn't worry to much about retaining the details as they are rarely included substantially in exam questions.
+
+The course is graded on a bucket system, where out of 1,080 possible total homework points, earning 900 points 
+fulfills 100% of the homework bucket. This allows for a generous margin of points to be lost before
+students cross any grade thresholds. (For exams, 360 points are available but only 300 maximum are counted towards the
+final grade)
+
+The Mathematical Foundations requirement can also be fulfilled by taking 2 of the following instead:
+- 21-259: Calculus in Three Dimensions
+- [21-241: Matrices and Linear Transformations](math_science_reqs/21241.md)
+- 21-254: Linear Algebra and Vector Calculus for Engineers
+- 21-260: Differential Equations
+
+## Prerequisites
+
+In order to take 18-202, students must first take 21-122: Integration and Approximation or have taken AP
+Calculus BC before attending Carnegie Mellon. It is also useful to be familiar with some linear algebra
+concepts before taking 18-202.
+
+## Homeworks
+
+Homeworks are assigned on a weekly basis, and length can vary. Each homework can include proofs, drill questions, 
+or conceptual questions. Students will be expected to use Python or MATLAB to carry out certain linear algebra/matrix operations.
+For those unfamiliar with the software, there is a TA-led bootcamp at the start of the semester.
+A good way to get a feel for how much work students are expected to show to earn full points is by
+working through problems in TA-led recitations. Homeworks can be conceptually difficult or plain
+tedious, so getting started early is recommended.
+
+## Quizzes
+
+Quizzes are offered during each lecture and typically ask simple questions relating to the most recent
+lecture topics. These are mostly used to make sure you are paying attention in lecture. Some quizzes
+offer bonus points for answering miscellaneous questions, though they do not count outside of the quiz
+bucket.
+
+## Exams
+
+18-202 has 3 Exams which account for 75% of a student's final grade. Students are allowed to bring
+a note sheet (up to 3 by the final exam), but the exam is otherwise closed-book.* Due to the note sheet,
```

Comment:
Do you mind doing the following for footnotes?

https://stackoverflow.com/questions/25579868/how-to-add-footnotes-to-github-flavoured-markdown

```
GitHub Flavored Markdown doesn't support footnotes, 
but you can manually fake it¹ with Unicode characters 
or superscript tags, e.g. <sup>1</sup>.
```